### PR TITLE
[stable/influxdb] Add the ingress docs and remove the maintainer who was no longer maintain the chart.

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: Aisuko <urakiny@gmail.com>

#### What this PR does / why we need it:

1, The latest Nginx ingress document address can make the user easily to add the annotation.
2, Remove the old maintainer who is no longer maintain the chart(This can let every PR can be assigned to the active maintainer). 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@jackzampolin are you ok with being removed?

**Comment by scottrigby** [comment]
More context: see [Slack chat](https://kubernetes.slack.com/archives/C6E3XH1ED/p1575305284333400?thread_ts=1574998850.327900&cid=C6E3XH1ED)

**Comment by naseemkullah** [comment]
If you would like an additional maintainer please feel free to add me @Aisuko, we have been using this chart in production at my org for quite a while.

**Comment by Aisuko** [comment]
@naseemkullah It's our pleasure you can be here, Sir. To be honest, I keep aiming to keep the chart more stable, powerful for the end-user and who new in deploying the application to Kubernetes. I'm happy to see more maintainers like you here. I will add you to the maintainer. Maybe, we can finish it 
 in this PR.

**Comment by Aisuko** [comment]
@naseemkullah Hello sir, sorry too late to update the status of this PR. I'd like to add you to the maintainer list, Would you mind told me your Email address?

**Comment by naseemkullah** [comment]
> @naseemkullah Hello sir, sorry so late to update the status of this PR. I'd like to add you to the maintainer list, Would you mind told me your Email address?

Sorry for the late reply:
naseem@transit.app

File: stable/influxdb/OWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
 approvers:
-- jackzampolin
 - aisuko
+- naseemkullah
 reviewers:
-- jackzampolin
 - aisuko
+- naseemkullah
```
Comment (naseemkullah) [inline stable/influxdb/OWNERS]: nit: add newline at EOF

File: stable/influxdb/values.yaml [hunk 2]
```diff
@@ -114,6 +114,11 @@ ingress:
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: influxdb.foobar.com
+
+  ## Ingress annotations done as key:value pairs
+  ## For a full list of possible ingress annotations, please see
+  ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md
```
Comment (naseemkullah) [inline stable/influxdb/values.yaml]: I don't think this paragraph is necessary imho. Furthermore the link is just for nginx-ingress annotations, not applicable to other ingress controllers.

**Review by naseemkullah — CHANGES_REQUESTED** [review]

File: stable/influxdb/values.yaml (hunk 2 above)
Comment (Aisuko) [reply stable/influxdb/values.yaml]: Thanks. Solved.

**Comment by naseemkullah** [comment]
@Aisuko can you /lgtm this please?

**Comment by Aisuko** [comment]
@naseemkullah There only the helm member can approve the PR. I do not have permission to approve myself PR.

**Comment by Aisuko** [comment]
The content was included in another PR. So, Close this one.

**Pull request closed without merging**